src/applications/vaos/api/index.js

```diff
@@ -434,7 +434,17 @@ export function updateAppointment(appt) {
 export function updateRequest(req) {
   let promise;
   if (USE_MOCK_DATA) {
-    promise = Promise.resolve();
+    promise = import('./requests.json')
+      .then(module => (module.default ? module.default : module))
+      .then(data => ({
+        data: {
+          id: req.id,
+          attributes: {
+            ...data.data.find(item => item.id === req.id).attributes,
+            status: 'Cancelled',
```

Comment:
Just curious why we're manually setting this status here.  Does the response not contain the updated `Cancelled` status?

Reply:
No, this is getting the original data from our mock data list, which wouldn't already have the "Cancelled" status.

Reply:
Oh.  Whoops, I overlooked that this was for the mock data, not a real api call.